server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java

```diff
@@ -58,11 +61,16 @@
     public SMTPMessageSender smtpMessageSender = new SMTPMessageSender(DOMAIN);
 
     @Override
-    protected GuiceJamesServer createJamesServer() {
-        return new GuiceJamesServer()
+    protected GuiceJamesServer createJamesServer() throws IOException {
+        String workingDir = temporaryFolder.newFolder().getAbsolutePath();
+        Configuration configuration = Configuration.builder()
+            .workingDirectory(workingDir)
```

Comment:
Passing a file as argument to `workingDirectory` seems legitimate. Can you add a method for that?